tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -612,7 +612,33 @@ final class TestingServer(
       (text, params) <- offsetParams(filename, query, root)
       highlights <- server.documentHighlights(params).asScala
     } yield {
-      TestHighlights.renderAsString(text, highlights.asScala.toList)
+      TestRanges.renderHighlightsAsString(text, highlights.asScala.toList)
+    }
+  }
+
+  def verifyImplementation(
```

Comment:
For consistency with other methods
```suggestion
  def assertImplementation(
```

Reply:
Changed to assert